modules/services/window-managers/i3.nix

```diff
@@ -410,6 +410,15 @@ let
         '';
       };
 
+      workspaceAutoBackAndForth = mkOption {
+        type = types.bool;
+        default = false;
+        example = true;
+        description = ''
+          Assume you are on workspace "1: www" and switch to "2: IM" using mod+2 because somebody sent you a message. You don’t need to remember where you came from now, you can just press $mod+2 again to switch back to "1: www".
```

Comment:
this is one long line. @rycee shouldn't the linter fail on this one ?

Reply:
I'm not sure how it's supposed to be formatted, but I've added some newlines 🙂

Reply:
@teto I'm not certain about how much the xml checks will look at line lengths. At the moment I'm more relying on the code reviewing for this 🙂